shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/core/util/ThreadUtil.java

```diff
@@ -37,4 +39,17 @@ public static void sleep(final long millis) {
         } catch (final InterruptedException ignored) {
         }
     }
+    
+    /**
+     * Sleep ignored InterruptedException.
+     *
+     * @param timeUnit time unit.
+     * @param number sleep time.
+     */
+    public static void sleep(final TimeUnit timeUnit, final int number) {
```

Comment:
`int number` could be `int timeout`, and it could be put before `TimeUnit timeUnit`.

`ThreadUtil.sleep(TimeUnit.SECONDS, 2)` vs `ThreadUtil.sleep(2, TimeUnit.SECONDS)`.